content/post/2022-11-18-global-leadership-team-transition/index.en.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: "Global Leadership Team Transition"
+author: "Shannon Pileggi"
+date: "2022-11-16"
+description: "After three years of service, Erin LeDell, Claudia Vitolo, and Hannah Frick are stepping down from the Global Leadership Team."
+tags: 
+- global
+- leadership 
+- team
+categories:
+- r-ladies
+---
+
+After three years of service on the R-Ladies Global Leadership Team, Erin LeDell 
+(United States), Claudia Vitolo (Italy), and Hannah Frick (England) are 
+stepping down. Under their leadership, R-Ladies survived and thrived throughout
+the pandemic by pivoting to online meet-ups, enabling knowledge sharing and community
+engagement both within and across chapters. During this time, R-Ladies experienced tremendous
+growth in community members and reach around the world. We are forever grateful
+to Erin, Claudia, and Hannah for steadfastly remaining at the helm during uncertain
+times, and for continuing to expand upon the foundation first established by Gabriela
+de Queiroz.
+
+We are pleased to share the new members of the Global Leadership Team: Athanasia 
+Monika Mowinckel (Norway), Yanina Bellini Saibene (Argentina), Shannon Pileggi
+(United States), Riva Quiroga (Chile), and Averi Giudicessi (United States). We are
+committed to the R-Ladies Global mission of encouraging, inspiring, and empowering
+people of minority genders in the R community, and we are honored to serve the 
+global organization in this capacity.
+
```

Comment:
We are not going to be adding any code chunks here, so I'd suggest we delete the Rmd an only edit in the pure md for this post. :)

Reply:
done!